hvplot/plotting/core.py

```diff
@@ -12,62 +14,95 @@
 
 
 class hvPlotBase:
+    """
+    The plotting method: `df.hvplot(...)` creates a plot similarly to the familiar Pandas
+    `df.plot` method.
+
+    For more detailed options use a specific ploting method, e.g. `df.hvplot.line`.
+
+    Reference: https://hvplot.holoviz.org/reference/index.html
+
+    Parameters
+    ----------
+    x : string, list, or array-like, optional
+        Field name(s) to draw x-positions from
+    y : string, list, or array-like, optional
+        Field name(s) to draw y-positions from
+    kind : string, optional
+        The kind of plot to generate, e.g. 'area', 'bar', 'line', 'scatter' etc. To see the
+        available plots run `print(df.hvplot.__all__)`.
+    **kwds : optional
+        Additional keywords arguments are documented in `hvplot.help('scatter')` or similar
+        depending on the type of plot.
+
+    Returns
+    -------
+    A Holoviews object. You can `print` the object to study its composition and run `hv.help` on
+    the object to learn more about its parameters and options.
+
+    Examples
+    --------
+
+    >>> import hvplot.pandas
+    >>> from bokeh.sampledata.degrees import data as deg
+    >>> line = deg.hvplot(
+    ...     x='Year', y=['Art and Performance', 'Business', 'Biology', 'Education', 'Computer Science'],
+    ...     value_label='% of Degrees Earned by Women', legend='top', height=500, width=620,
+    ...     kind="line",
+    ... )
+    >>> line
+
+    You can can add *markers* to a `line` plot by overlaying with a `scatter` plot.
+
+    >>> scatter = deg.hvplot.scatter(
+    ...     x='Year', y=['Art and Performance', 'Business', 'Biology', 'Education', 'Computer Science'],
+    ...     value_label='% of Degrees Earned by Women', legend='top', height=500, width=620,
+    ...     kind='scatter',
+    ... )
+    >>> line * scatter
+
+    Please note that you can pass widgets or reactive functions as arguments instead of
+    literal values, c.f. https://hvplot.holoviz.org/user_guide/Widgets.html.
+    """
 
     __all__ = []
 
     def __init__(self, data, custom_plots={}, **metadata):
-        if 'query' in metadata:
-            data = data.query(metadata.pop('query'))
-        if 'sel' in metadata:
-            data = data.sel(**metadata.pop('sel'))
-        if 'isel' in metadata:
-            data = data.isel(**metadata.pop('isel'))
+        "Hello"
+        if "query" in metadata:
+            data = data.query(metadata.pop("query"))
+        if "sel" in metadata:
+            data = data.sel(**metadata.pop("sel"))
+        if "isel" in metadata:
+            data = data.isel(**metadata.pop("isel"))
```

Comment:
Oh be careful it seems your autoformatter was busy doing some extra work!

Reply:
Yes. I got a new way of working (cloud instead of laptop) and setup VS Code to autoformat on save using `black`. Will turn it off again.

Reply:
But could we start using `isort` and `black` soon on these projects. Just makes your life as a developer so much easier.